### ISSUE TITLE
Why do I need FileReader at all?

### ISSUE THREAD

**Issue by sunshineo** [danialfarid/ng-file-upload#624]
I need to post without using multipart
I can do this
```
$http.post("https://api.parse.com/1/files/" + files[0].name, files[0], {
		            withCredentials: false,
		            headers: {
		            		'Content-Type': files[0].type
		            },
		           transformRequest: angular.identity
		    });
```
It works totally fine. But when I try to switch to this library $uploader.http() method based on https://github.com/danialfarid/ng-file-upload/issues/88 I have to use a FileReader. 

What is going on? Why do I suddenly need FileReader?

**Comment by danialfarid** [comment]
You are right it is not needed.
I have updated the README and gonna update the demo page. 

Thanks

**Comment by danialfarid** [comment]
Fixed in version 3.3.0
